backend/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3
```

Comment:
Vim, ze zrovna u Pythonu je to obcas problem, ale bylo by super vytvorit ten build nad nejakou produkcni image. Klidne i dvoufazovou pro "build" a potom beh. Je to napriklad dulezite co se velikosti te vysledne image tyce (misto v repozitari a cena za prenesena data) a tahle image neprojde checkem na ECR (je v terraformu nastavene checkovani tech prichozich image a tyhle budou hodne cervene.

Ja pouzivam v produkci bezne Distroless nebo trochu hure Alpine verze.

Reply:
Jo, tam je určitě prostor pro optimalizaci, ale to bych udělal v rámci nového PR, ať toto můžou kluci začít používat.